src/main.js

```diff
@@ -165,7 +165,7 @@ app.post('/dev/login', withConfig, async (req, res, next) => {
   const fakeUsername = cleaned.fakeUsername.length === 0 ? undefined : cleaned.fakeUsername
   const payload = {'eraCommonsUsername': fakeUsername}
   const privateKey = req.config.data.devKeyPrivate
-  const token = jwt.sign(payload, privateKey, {algorithm: 'RS256'})
+  const token = jwt.sign(payload, privateKey, {algorithm: 'RS256', expiresIn: '60s'})
```

Comment:
Let's make this lenient since we don't have visibility into all use cases.
```suggestion
  const token = jwt.sign(payload, privateKey, {algorithm: 'RS256', expiresIn: '1h'})
```